modules/client-configuration/pages/autoyast.adoc

```diff
@@ -1,20 +1,18 @@
 [[client-cfg-autoinstallation-autoyast]]
-= {ay}
+= AutoYaSt
 
+When you install a {sle} client, there are a number of questions you need to answer.
+In order to automate installation, you can create an {ay} file with all the answers to those questions, so that no user intervention is required.
```

Comment:
```suggestion
To automate installation, you can create an {ay} file with all the answers to those questions, so that no user intervention is required.
```